Source/WebCore/crypto/cocoa/CryptoAlgorithmAESKWMac.mm

```diff
@@ -42,6 +43,17 @@ static ExceptionOr<Vector<uint8_t>> wrapKeyAESKW(const Vector<uint8_t>& key, con
     return WTFMove(result);
 }
 
+static ExceptionOr<Vector<uint8_t>> wrapKeyAESKWCryptoKit(const Vector<uint8_t>& key, const Vector<uint8_t>& data)
+{
+    // CryptoKit has no way to return the size so assuming 2x input size.
```

Comment:
Is there some mathematical property of AES keys that proves that this is sufficient?  If it is not sufficient, should we retry with more output size?

Reply:
Have used the RFC spec now to choose a more educated size.